docs/introduction/entity-component-system.md

```diff
@@ -343,15 +343,21 @@ to look at include:
 - [A-Painter](https://github.com/aframevr/a-painter/)
 - [A-Blast](https://github.com/aframevr/a-blast/)
 
-#### *A Week of A-Frame*
+#### A-Frame Blog
 
 [blog]: https://aframe.io/blog/
 [homepage]: https://aframe.io/
 
-[Every week on the blog][blog], we round up all the activity in the A-Frame
-community and ecosystem. This includes featuring components that have been
-freshly released or updated. [The homepage of A-Frame][homepage] will usually
-have a link to the most recent *A Week of A-Frame* entry.
+Until mid-2019, there was a weekly post on the [the blog][blog], "A week of A-Frame" which rounded up all the activity in the A-Frame
```

Comment:
Not sure if the explanation is necessary. We can just perhaps get rid of the `every week blog` reference all together

Reply:
Not sure if I understood your intent here correctly, but I have cut the text back considerably.